### PR TITLE
Change route for dashboard to fit user story

### PR DESCRIPTION
**Pull request by hashmaster3k**
### Description:

  Changes URI route for dashboard to better fit user story. User story requests /dashboard after URL. Initially had it as /user/dashboard but now changed to /dashboard

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -6,7 +6,5 @@
 
   post '/login', to: 'sessions#create'
 
-  namespace :user do
-    get '/dashboard', to: 'dashboard#index'
-  end
```
Comment (hashmaster3k) [inline config/routes.rb:11]: Removed namespace due to only needing module and not scope.

File: config/routes.rb [hunk 2]
```diff
@@ -6,7 +6,5 @@
...
+  get '/dashboard', to: 'user/dashboard#index'
```
Comment (hashmaster3k) [inline config/routes.rb:9]: Cleaner and only what we need.

**Review by nicorithner — APPROVED** [review]
Thank you for working on this

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -16,7 +16,7 @@ def create_user
     user = User.new(user_params)
     if user.save
       session[:user_id] = user.id
-      redirect_to '/user/dashboard'
+      redirect_to dashboard_path
```
Comment (hashmaster3k) [inline app/controllers/users_controller.rb:19]: Went thought and changed all the pathing.

**Comment by hashmaster3k** [comment]
Test still at 100% and RuboCop has no concerns.

File: config/routes.rb (hunk 2 above)
Comment (nicorithner) [reply config/routes.rb:9]: This looks much cleaner

**Pull request merged**